eiffel-vocabulary/EiffelTestCaseFinishedEvent.md

```diff
@@ -86,7 +86,7 @@ __Description:__ The URI at which the log can be retrieved.
 ## Version History
 | Version   | Introduced in                                          | Changes                                 |
 | --------- | ------------------------------------------------------ | --------------------------------------- |
-| 1.0.0     | [edition-bordeaux](../../../tree/edition-bordeaux)     | Initial version.                        |
+| 1.0.1     | [edition-bordeaux](../../../tree/edition-bordeaux)     | Current version.                        |
```

Comment:
Sorry, I didn't spot this one before. This should be a history of all versions, not just the latest. In other words,
`| 1.0.1     | Current version | data.outcome.metrics.value and data.outcome.metrics.name made mandatory. |`
`| 1.0.0     | [edition-bordeaux](../../../tree/edition-bordeaux)     | Initial version.                        |`